### DISCUSSION TITLE
Is there a way to keep the bundlesize small

### DISCUSSION THREAD

**Discussion by 0xAlwaysDumpling** [chakra-ui/chakra-ui#7332] [Q&A]
Currently using 1.7.1 and looking to bump package versions. But I noticed the latest package is 2x the size. Anyway I can maintain a smaller bundle size?
<img width="1354" alt="Screen Shot 2023-02-08 at 8 53 56 AM" src="https://user-images.githubusercontent.com/8901270/217551250-c922048a-5883-4041-abe2-41faea6a3a0e.png">

**Comment by galliotl** [comment]
I asked the same question #7234 it seems that it has something to do with a recent PR that sets a new target for javascript transpilation (if I understood correctly...)
